articles/stream-analytics/stream-analytics-with-azure-functions.md

```diff
@@ -194,6 +194,13 @@ Follow the [Real-time fraud detection](stream-analytics-real-time-fraud-detectio
    This command should print the value for the specified key:
 
    ![Screenshot of Azure Redis Cache output](./media/stream-analytics-with-azure-functions/image5.png)
+   
+## Error Handling and Retires
```

Comment:
Typo - retires. Should be retries. 
Please use sentence casing. Lowercase handling and retries. 
> ## Error handling and retries